rcl/include/rcl/logging.h

```diff
@@ -71,6 +71,25 @@ RCL_PUBLIC
 RCL_WARN_UNUSED
 rcl_ret_t rcl_logging_fini();
 
+/// See if logging rosout is enabled.
+/**
+ * This function is meant to be used to check if logging rosout is enabled
```

Comment:
nitpick: use punctuation

```suggestion
 * This function is meant to be used to check if logging rosout is enabled.
```

Reply:
thanks for pointing this out, done with https://github.com/ros2/rcl/pull/514/commits/2cefd98952f8bbeef1e8a57ec5730e9402e5c925